chapters/introduction.tex

```diff
@@ -72,8 +72,33 @@ \section{Some Definitions}\label{some-definitions}
 and components, and generation of connection equations from \lstinline!connect!-equations (basically, mapping the hierarchical structure of a model into a set of differential, algebraic and discrete equations together with the corresponding variable declarations and function definitions from the model).
 \end{definition}
 
+\begin{definition}[Initialization]\index{initialization}
+The simulation starts with solving the initialization problem at the starting time giving values for all variables that are consistent with the result of the flattening.
+\end{definition}
+
+\begin{definition}[Transient analysis]\index{transient analysis}\index{initial value problem!see{transient analysis}}
+Starting from the result of the initialization problem the model is then simulated forward in time.
+This generates values for the variables at different times -- and uses numerical methods for handling the hybrid DAE generated by the flattening.
+\end{definition}
+
+\begin{definition}[Simulation]\index{simulation}
+For a model simulation is the combination of the initialization followed by the transient analysis.
```

Comment:
I suppose we could let the condition on applying to a simulation model be implicit:
```suggestion
Simulation is the combination of solving the initialization problem and carrying out transient analysis.
```

Reply:
I will apply the first change, but as previously noted defining 'Initialization' means that I avoid the other changes.

Reply:
Sure, let's try to keep down use of _initialization problem_.  Resolving.